tools/automatic_query_fixer/src/main/java/com/google/cloud/bigquery/utils/queryfixer/entity/TokenImpl.java

```diff
@@ -13,8 +13,11 @@ public TokenImpl (Token token) {
     this.token = token;
   }
 
-  @Override public int getKind() {
-    return token.kind;
+  @Override public Kind getKind() {
+    if (token.kind == 788) {
```

Comment:
788? i think it should corresponding to the index defined in the kind enum?

Reply:
Calcite defines hundreds of token kinds, so number 788 indeed means an ordinary identifier. Anyway this token class will be deprecated very soon once the ZetaSQL server is setup and a ZetaSQL token is introduced.